public/js/game.js

```diff
@@ -38,10 +41,19 @@ let getLevelData = level => {
     // Starting score
     score = 100;
     scoreDisplay.textContent = score;
-    let scoreTimer = setInterval(() => {
-      score--;
-      scoreDisplay.textContent = score;
-    }, 100);
+    scoreTimer = setInterval(() => {
+      if (score > 0) {
+        score--;
+        scoreDisplay.textContent = score;
+      }
+    }, 200);
+    // Starting time
+    let startTime = Date.now();
+    timeTimer = setInterval(() => {
+      time = Date.now() - startTime;
+      let date = new Date(time);
+      timeDisplay.textContent = date.getSeconds();
+    }, 200);
```

Comment:
So this is counting down every 200 ms, but didn't we talk about counting down every second?

Reply:
I don't recall setting a time to decrement the score.  This works pretty well for a game timer.  If the score goes down every 200ms, then its down 5 points in 1 second, still leaving 20 seconds in a level before it hits zero.

Reply:
Alrighty that sounds good. 20 seconds does sound like a reasonable amount of time.